src/Accordion/index.js

```diff
@@ -1,2 +1 @@
-export { default } from './Accordion';
-export { default as AccordionSection } from './AccordionSection';
+export { default as AccordionSection } from './AccordionSection'; // eslint-disable-line import/prefer-default-export
```

Comment:
Since there is no longer an Accordion should all the references and files be updated to AccordionSection? We could then move it all from the Accordion folder to the AccordionSection folder.

Reply:
Yeah, you're probably right. This should be moved to `AccordionSection`

Reply:
Updated!